PiwikTracker/Queue.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+/// A queue in which Objects that conform to queueable can be stored.
```

Comment:
Nitpicking, I think you can skip much of the comments and api documentation for internal classes. I would only add api documentation to types and methods visible outside the framework.

It is pretty clear what a queue does and the method names are self explanatory.

Comments have a tendency to root and become lies after awhile (already now this comment claim object must conform to queueable which is no longer true :)